src/base/settingvalue.h

```diff
@@ -53,18 +60,18 @@ class CachedSettingValue
         return m_value;
     }
 
+    operator T() const
+    {
+        return m_value;
```

Comment:
What's the reason for this change? IMO, it is better to have only a single function that access member variable.